src/common/config/store/driver/http.go

```diff
@@ -0,0 +1,29 @@
+package driver
```

Comment:
is the http a driver? should it not a store?

Reply:
it is a driver